src/cart/crt_init.c

```diff
@@ -90,6 +53,9 @@ crt_lib_init(void)
 	crt_gdata.cg_rpcid = start_rpcid;
 	crt_gdata.cg_num_cores = sysconf(_SC_NPROCESSORS_ONLN);
 	crt_gdata.cg_iv_inline_limit = 19456; /* 19KB */
+
+	/* envs not inited until crt_init() time */
+	memset(&g_envs, 0x0, sizeof(struct crt_envs_t));
```

Comment:
you could do `sizeof(g_envs)` to not worry about the type.

Reply:
ok